package.json

```diff
@@ -130,6 +131,7 @@
     "syncpack": "^4.3.1",
     "through2": "^2.0.3",
     "tmp": "^0.0.33",
+    "ts-jest": "^24.0.2",
```

Comment:
used to transpile code of jest tests - should be removed after fixes for Babel config will be merged @layershifter